README.md

```diff
@@ -106,6 +106,14 @@ are still up-to-date.
 
 To build a GHC source distribution tarball, run Hadrian with the `sdist-ghc` target.
 
+#### Installation
+
+To install GHC artifacts, first build everything, then run the `install` target.
```

Comment:
That's a great point -- but this is only partially supported now. I should fix it later